src/Resources/contao/dca/tl_module.php

```diff
@@ -9,6 +9,8 @@
  * @license http://www.gnu.org/licences/lgpl-3.0.html LGPL
  */
 
+\Contao\System::loadLanguageFile('tl_module');
```

Comment:
I'm not sure if this is the right solution - but otherwise there are some warnings for missing translation keys!